Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -94,6 +94,7 @@ instance Measure.instFunLike [MeasurableSpace α] : FunLike (Measure α) (Set α
 
 #noalign measure_theory.measure.has_coe_to_fun
 
+set_option linter.deprecated false in -- Not immediately obvious how to use `measure_empty` here.
```

Comment:
Maybe something like:
```suggestion
#adaptation_note /-- (4.9.0) Not immediately obvious how to use `measure_empty` here. -/
set_option linter.deprecated false in
```
Please rephrase!

Reply:
I disagree with using `#adaptation_note` here. This has nothing to do with v4.9.0: it's always been deprecated, the only new thing is that v4.9.0 will tell us about this.

Reply:
I think `set_option linter.deprecated false` is exactly the annotation we need to keep track of these defects!